client/src/pages/OnBoard.js

```diff
@@ -0,0 +1,226 @@
+import React, { useState } from 'react';
+//Material-ui imports
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Background from '../elements/Background';
+import IconButton from '@material-ui/core/IconButton';
+import RemoveCircleIcon from '@material-ui/icons/RemoveCircle';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(12),
+    padding: theme.spacing(5),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center'
+  },
+  headerText: {
+    marginTop: theme.spacing(10)
+  },
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    textAlign: 'center'
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+    padding: theme.spacing(1, 6),
+    border: 0,
+    borderRadius: 25,
+    color: '#fff',
+    textTransform: 'none',
+    fontSize: 15
+  },
+  add: {
+    margin: theme.spacing(3, 0, 2),
+    padding: theme.spacing(1, 6),
+    border: 0,
+    borderRadius: 25,
+    textTransform: 'none',
+    fontSize: 16
+  },
+  removeIcon: {
+    color: theme.palette.button.remove
+  }
+}));
+
+export default function OnBoard() {
+  const classes = useStyles();
+
+  const languages = [ 'Python', 'JavaScript', 'Java', 'PHP', 'GoLang', 'C#', 'C++', 'Ruby' ];
+  const levels = [ 'Beginner', 'Advanced', 'Expert' ];
+
+  const [ skillList, setSkillList ] = useState([ { language: '', level: '' } ]);
+  const [ submitClicked, setSubmitClicked ] = useState(false);
+
```

Comment:
This is fine for now, but we should move to an action creator that puts the app in a loading state when submit is clicked. This way components can be consistently controlled, ie `state.loading ? disabled : enabled`